### ISSUE TITLE
Adding accordion sub-menu

### ISSUE THREAD

**Issue by LindsayYoung** [18F/openFEC#898]
Right now we are not showing subtotals and the FEC users, think things are broken/missing.

@noahmanger and I talked about this before and wanted to implement expandable sub-totals.

For example, you would open Receipts and see total contributions but that would expand to include the different kind of contributions. In there, individual contributions would expand to itemized and unitemized.  

 Also, @noahmanger Just wanted to make sure you understood that there would be subtotals in the subtotals and those should be collapsable too. If there are a lot of clicks, we might want an expand all option or something. 

from #884

**Comment by noahmanger** [comment]
Yeah this is exactly why we punted on including them in the first place ;) . 

I think I like the idea of having sub-totals rows just be larger to show the sub-sub-totals, without having to click to expand. I think I should take a stab at designing how this would look, but in the mean time I'd say those sub-sub-totals can just be listed in a div nested in the sub-total row.

**Comment by LindsayYoung** [comment]
Ok, if I understand you right-  in the following example total contributions would expand and that would show all the other things in one step. I like that better. 
![screen shot 2015-06-02 at 5 41 14 pm](https://cloud.githubusercontent.com/assets/4406333/7947975/98e7dcc8-094e-11e5-8274-96f9ee9d8f52.png)

**Comment by LindsayYoung** [comment]
Also, reversing this commit should add back the subtotals
https://github.com/18F/openFEC-web-app/commit/df929ebe56af542cc04075c3ce976abcf92ca399

**Comment by LindsayYoung** [comment]
I can add back in the subtotals.

**Comment by LindsayYoung** [comment]
I am, breaking the adding totals part into a separate issue.

accordion work depends on 
depends on #937

**Comment by LindsayYoung** [comment]
The subtotals are back in.

**Comment by noahmanger** [comment]
To be clear, the sub-totals are in but not displaying in the nested way
like I designed, right?

On Wed, Jun 10, 2015 at 8:01 AM, Lindsay Young <notifications@github.com>
wrote:

> The subtotals are back in.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/18F/openFEC/issues/898#issuecomment-110786697>.
>



-- 
Noah Manger
18F <http://18f.gsa.gov> | Work: 415-696-6146

**Comment by LindsayYoung** [comment]
Correct, that is what this issue is for.

**Comment by noahmanger** [comment]
Here's mocks: 

![option 1](https://cloud.githubusercontent.com/assets/1696495/8011699/b5e08d84-0b6f-11e5-909f-37f26ceb113e.png)

In option 2, we just show bars for each sub-sub-total but not the sub-total. 

![option 2](https://cloud.githubusercontent.com/assets/1696495/8011702/c155ba7c-0b6f-11e5-8fe6-f09e81e4a55e.png)

**Comment by emileighoutlaw** [comment]
@noahmanger popping in for a grammar thing: Based on the way you've got capitalization here, I think you should have "Coverage" in "Report Coverage" capitalized!

**Comment by noahmanger** [comment]
I'm going to break this out into a new issue: https://github.com/18F/openFEC/issues/1280
